### ISSUE TITLE
dtc should parse or ignore cpp line info

### ISSUE THREAD

**Issue by zxombie** [davidchisnall/dtc#3]
The dts may contain output from cpp to indicate the source line and file. dtc should either parse this, or ignore it.

An example of this would be:
```
# 1 "src/arm64/al/alpine-v2-evp.dts"
# 1 "<built-in>" 1
  1 "src/arm64/al/alpine-v2-evp.dts" 2
# 35 "src/arm64/al/alpine-v2-evp.dts"
# 1 "src/arm64/al/alpine-v2.dtsi" 1
# 35 "src/arm64/al/alpine-v2.dtsi"
/dts-v1/;
```
This causes dtc to complain with:
```
Error on line 1: Expected /dts-v1/; version string
# 1 "src/arm64/al/alpine-v2-evp.dts"
^
Error on line 1: Failed to find root node /.
# 1 "src/arm64/al/alpine-v2-evp.dts"
^
Error on line 1: Failed to find root node /.
# 1 "src/arm64/al/alpine-v2-evp.dts"
^
Failed to parse tree.  Unhappy face!
```

**Comment by davidchisnall** [comment]
Is line 3 in the example intentionally missing the leading #?

**Comment by zxombie** [comment]
I think is was an issue with my terminal, it seems to be there when I regenerate the file.

**Comment by davidchisnall** [comment]
Fixed on 6b363d58e
